<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlayerView</class>
 <widget class="QMainWindow" name="PlayerView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>879</width>
    <height>535</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>VLCPlayer</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="1">
     <widget class="QPushButton" name="btnPlay">
      <property name="text">
       <string>Play</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QPushButton" name="btnPause">
      <property name="text">
       <string>Pause</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="btnOpenVideo">
      <property name="text">
       <string>Open Video</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="3">
     <widget class="CRenderWidget" name="RenderWidget"/>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>CRenderWidget</class>
   <extends>QOpenGLWidget</extends>
   <header location="global">RenderWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
